types/jasmine/index.d.ts

```diff
@@ -858,14 +858,60 @@ declare namespace jasmine {
     interface PassedExpectation extends CustomReportExpectation {
     }
 
+    interface DeprecatedExpectation {
+        message: string;
+    }
+
     interface CustomReporterResult {
+        /**
+         * The unique id of this spec.
+         */
+        id: string;
```

Comment:
Isn't this an integer?

Reply:
It is typed as `{Int}` in the JSDoc... but it's actually always a string x.x